packs/jira/actions/create_issue.py

```diff
@@ -7,8 +7,11 @@
 
 
 class CreateJiraIssueAction(BaseJiraAction):
-    def run(self, summary, type, description=None):
-        project = self.config['project']
+    def run(self, summary, type, description=None, project=None):
+        if project:
+            project = project
```

Comment:
project = project or self.config['project']

I guess that is more pythonic or some such thing.